src/DynamoCoreWpf/ViewModels/Watch3D/DefaultWatch3DViewModel.cs

```diff
@@ -196,7 +198,7 @@ public DefaultWatch3DViewModel(Watch3DViewModelStartupParams parameters)
             TogglePanCommand = new DelegateCommand(TogglePan, CanTogglePan);
             ToggleOrbitCommand = new DelegateCommand(ToggleOrbit, CanToggleOrbit);
             ToggleCanNavigateBackgroundCommand = new DelegateCommand(ToggleCanNavigateBackground, CanToggleCanNavigateBackground);
-
+            ZoomToFitCommand = new DelegateCommand(ZoomToFit, CanZoomToFit);
```

Comment:
ZoomToFit is now a command on the view model instead of a click event handler in the view's code behind.